notokenheader.txt

```diff
@@ -0,0 +1 @@
+Accept: application/json
```

Comment:
also: does this need to be in the root of the muesli project?

Reply:
Also: No.
It is there because you are reviewing a WIP branch. Will be removed.